forge/ee/routes/teamBroker/index.js

```diff
@@ -0,0 +1,299 @@
+                    acls: { type: 'array' },
+                    username: { type: 'string' },
+                    password: { type: 'string' }
+                }
+            },
+            response: {
+                201: {
+                    type: 'object',
+                    properties: {
+                        id: { type: 'string' },
+                        username: { type: 'string' },
+                        acls: { type: 'array' }
+                    }
+                },
+                '4xx': {
+                    $ref: 'APIError'
+                },
+                500: {
+                    $ref: 'APIError'
+                }
+            }
+        }
+    }, async (request, reply) => {
+        try {
+            const newUser = request.body
+            newUser.acls = JSON.stringify(newUser.acls)
+            await request.team.checkTeamBrokerClientCreateAllowed()
+            const user = await app.db.models.TeamBrokerClient.create({ ...request.body, TeamId: request.team.id })
+            reply.status(201).send(app.db.views.TeamBrokerClient.user(user))
+        } catch (err) {
+            return reply
+                .code(err.statusCode || 400)
+                .send({
+                    code: err.code || 'unknown_error',
+                    error: err.error || err.message
+                })
+        }
+    })
+
+    /**
+     * Get a specific MQTT Client
+     * @name /api/v1/teams/:teamId/broker/client/:username
+     * @static
+     * @memberof forge.routes.api.team.broker
+     */
+    app.get('/client/:username', {
+        schema: {
+            summary: 'Get details about a specific MQTT client',
+            tags: ['MQTT Broker'],
+            params: {
+                type: 'object',
+                properties: {
+                    teamId: { type: 'string' },
+                    username: { type: 'string' }
+                }
+            },
+            response: {
+                200: {
+                    type: 'object',
+                    properties: {
+                        id: { type: 'string' },
+                        username: { type: 'string' },
+                        acls: { type: 'array' }
+                    }
+                },
+                '4xx': {
+                    $ref: 'APIError'
+                },
+                500: {
+                    $ref: 'APIError'
+                }
+            }
+        }
+    }, async (request, reply) => {
+        const user = await app.db.models.TeamBrokerClient.byUsername(request.params.username, request.team.hashid)
+        if (user) {
+            reply.send(app.db.views.TeamBrokerClient.user(user))
+        } else {
+            reply.status(404).send({})
+        }
+    })
+
+    /**
+     * Update a specific MQTT Client
+     * @name /api/v1/teams/:teamId/broker/client/:username
+     * @static
+     * @memberof forge.routes.api.team.broker
+     */
+    app.patch('/client/:username', {
```

Comment:
```suggestion
    app.put('/client/:username', {
```

Put not Patch for updates like this